src/AWSS3.jl

```diff
@@ -7,9 +7,6 @@
 #==============================================================================#
 
 
-__precompile__()
```

Comment:
Any reason to take this out?

Reply:
It has been unnecessary (precompile true by default, same as `__precompile__()` since Julia version 0.7.  Since the compat bound is 1 anyway, it is not necessary on any version of Julia that this package will work on.  Seemed superfluous.

Reply:
Ah interesting, didn't know that!